### PR TITLE
Use new numpy pinning

### PR DESCRIPTION
**Pull request by kain88-de**
[doc](https://conda-forge.org/docs/meta.html#building-against-numpy)

@ocefpaf Please tell me if I made mistakes setting up the CI for the new changes.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -9,9 +9,10 @@ osx_image: xcode6.4
 env:
   matrix:
     
-    - CONDA_NPY=113  CONDA_PY=27
-    - CONDA_NPY=113  CONDA_PY=35
-    - CONDA_NPY=113  CONDA_PY=36
+    - CONDA_NPY=107  CONDA_PY=27
+    - CONDA_NPY=109  CONDA_PY=35
+    - CONDA_NPY=109  CONDA_PY=34
+    - CONDA_NPY=111  CONDA_PY=36
```
Comment (ocefpaf) [inline .travis.yml]: That is odd. We should lose the `numpy` here. Did conda-smithy do that?

Comment (kain88-de) [inline .travis.yml]: Nope I did that by hand.

Comment (kain88-de) [inline .travis.yml]: So the numpy settings should be removed?

Comment (ocefpaf) [inline .travis.yml]: Yep. `conda-smithy` will do the right thing and "outsource" the `numpy` selection to `conda` via the different build/run pinning.

We are looking into doing the same for all the pinnings in conda-forge BTW, but this one was first b/c it reflects in the CIs directly. Other others will alleviate incompatibilities. The rule of thumb is to  build with the oldest version possible and run with the latest version that is backwards compatible. When a new version breaks the `ABI` we need to choose to use it as the base to build or take some time to adopt it. That choice is better than the current pinning scheme that we have which cause many breakages with `defaults`.

**Comment by ocefpaf** [comment]
@kain88-de see this PR that I just sent https://github.com/conda-forge/pyspharm-feedstock/pull/8
The second commit is the result of a re-rendering with `conda-smithy`, which removed all the `numpy` stuff from the build matrix.

**Comment by kain88-de** [comment]
@ocefpaf I moved the changes in `meta.yaml` to one commit and made a clean rerender. conda-smithy might have done something weird though. It deleted the travis configuration.

**Comment by ocefpaf** [comment]
Let me try this with my "hacked" `conda-smithy`. Maybe the latest version needs some attention. (There is the CircleCI 2.0 stuff in there.)

**Comment by kain88-de** [comment]
I'll also check the lastest version. I just see that I'm one minor release behind.

**Comment by ocefpaf** [comment]
> I'll also check the lastest version. I just see that I'm one minor release behind.

I found the problem. It is in the `skip` statement, see #57.

**Pull request closed without merging**